src/main/java/com/aviva/DataProcess/CarDataProcessingInterface.java

```diff
@@ -0,0 +1,39 @@
+package com.aviva.DataProcess;
+
+import com.aviva.Entities.Car;
+
+import java.util.ArrayList;
+
+/**
+ * This interface defines methods that process data from our database into
+ * a format that our business rules can use.
+ *
+ * Implementations of this interface should NOT talk directly to the SQL db
+ */
+public interface CarDataProcessingInterface {
+    /**
+     * Iterates through our table of cars and returns all of them
+     * @return a list of car objects from our database sorted by ascending price
+     */
+    static ArrayList<Car> getAllCars(){
+        return null;
+    }
+
+    /**
+     * @param name Full label of car as YEAR MAKE MODEL
+     * @return a Car object using the database
+     */
+    static Car getCarByName(String name){
+        return null;
+    }
+
+
+
+    // Not sure what this means/requires more discussion
```

Comment:
This method shouldnt be that complicated because ResultSet already has a next() attribute you can use